File: Lecture-1/Code/MinSort/MinSort.java
void minSort(int[] lst) {
	for (int i = 0; i < lst.length; i++) {
		int min = i;

		for (int j = i + 1; j < lst.length; j++)
			if (lst[j] < lst[min])
				min = j;

		if (min != i) {
			int tmp = lst[i];
			lst[i] = lst[min];
			lst[min] = tmp;
		}
	}
}

File: Lecture-2/Code/MinSort/MinSort.py
def minsort(lst):
	for i in range(0, len(lst)-1):
		minimum = i

		for j in range(i+1, len(lst)):
			if lst[j] < lst[minimum]:
				minimum = j

		if minimum != i:
			lst[i], lst[minimum] = \
				lst[minimum], lst[i]

	return lst
